admin/scss/_actionTabs.scss

```diff
@@ -170,7 +170,7 @@ $border: 1px solid darken(#D9D9D9, 15%);
 					>li{
 						width: auto;
 						a.tab-nav-link{
-							@include box-sizing(border-box);
+							box-sizin: border-box;
 							@include duration(0.5s);
```

Comment:
Could this be handled by autoprefixer too with just `transition-duration`? It looks to me like the `transition` and `duration` mixins in admin/scss/_mixins.scss could be removed now - `transition()` doesn't seem to be used at all and `duration()` is only used here a couple of times, without the second argument. Not a must-have, just seems like a good a time as any to include :D